lib/realtime/application.ex

```diff
@@ -45,6 +45,12 @@ defmodule Realtime.Application do
     :syn.add_node_to_scopes([:users, RegionNodes])
     :syn.join(RegionNodes, System.get_env("FLY_REGION"), self(), node: node())
 
+    {:ok, modules} = :application.get_key(:realtime, :modules)
+
+    modules
+    |> Enum.filter(&String.starts_with?(to_string(&1), "Elixir.Realtime.Tenants.Repo.Migrations"))
+    |> Enum.each(&Code.ensure_loaded!/1)
+
```

Comment:
let's move this to `Extensions.PostgresCdcRls.Supervisor`

Reply:
done (https://github.com/supabase/realtime/pull/520/files#diff-1de8846d5d70df4b816a1b2bca51468d6c8386bc81f9efe82df2bf837367497d)